python/sdk_e2e/test_track_1/conftest.py

```diff
@@ -244,7 +244,7 @@ def pnp_model_id():
 @pytest.fixture(scope="session")
 def random_key_factory():
     def factory_function():
-        return "prop{}".format(random.choice([1, 3, 4, 5]))
+        return "prop{}".format(random.choice([4]))
```

Comment:
Isn't this equivalent to just doing `return "prop4"`?

Reply:
Yes, you are correct.  The property tests are very unreliable right now.  This is a temporary workaround that didn't actually work around anything, and I think I have a real fix in my my mental queue.